### PR TITLE
Add signup function

### PR DESCRIPTION
**Pull request by yuya-minamide**
## Proposed Changes
- This indicates adding signup and login functionality
#### Code changes
-Added-
- login folder
- signup folder
- firebase.ts

#### Issues affected
- #72

## Additional Info
- I tried to separate the HTML part of the login page from the Logic part, but it's not done.
- I need to add log out function

## Screenshots and/or video
- None

## Testing Plan
- [x] Check that information is registered in firebase
- [x] Can go to the home page when the correct information is entered

#### Basics
- [x] Local QA is complete
- [x] No debugging `console.log()` statements
- [x] Commented-out code removed
- [x] Unused code removed (imports, functions, styles, etc - if it's not used anywhere, it's not in this PR)

### CODE REVIEW

File: src/hooks/firebase.ts [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  storageBucket: process.env.NEXT_PUBLIC_STORAGE_BUCKET,
+  messagingSenderId: process.env.NEXT_PUBLIC_MESSAGING_SENDER_ID,
+  appId: process.env.NEXT_PUBLIC_APP_ID,
+}
+
+const app = initializeApp(firebaseConfig)
+const auth = getAuth(app)
+export function useAuth() {
+  return auth
+}
```
Comment (Soma-dev0808) [inline src/hooks/firebase.ts]: ここはいらないです！

File: .env.local-sample [hunk 2]
```diff
@@ -0,0 +1,6 @@
+NEXT_PUBLIC_API_KEY=
+NEXT_PUBLIC_AUTH_DOMAIN=
+NEXT_PUBLIC_PROJECT_ID=
+NEXT_PUBLIC_STORAGE_BUCKET=
+NEXT_PUBLIC_MESSAGING_SENDER_ID=
+NEXT_PUBLIC_APP_ID=
```
Comment (Soma-dev0808) [inline .env.local-sample]: envファイルは含まないようにお願いします！
@yuya-minamide

**Review by Soma-dev0808 — CHANGES_REQUESTED** [review]

File: src/hooks/firebase.ts (hunk 1 above)
Comment (Soma-dev0808) [reply src/hooks/firebase.ts]: ここは
https://github.com/HidemichiShimura/cafe-drink-customizer/pull/80/commits/dce7a4966c27f298f8921a648475b35c8ce8fdf3
で修正しています！

Comment (yuya-minamide) [reply src/hooks/firebase.ts]: 了解です。
ありがとうございます！

**Pull request merged**